packages/react-ui-components/src/DateInput/dateInput.js

```diff
@@ -3,7 +3,12 @@ import PropTypes from 'prop-types';
 import enhanceWithClickOutside from 'react-click-outside';
 import moment from 'moment';
 import mergeClassNames from 'classnames';
+import {connect} from 'react-redux';
+import {$transform, $get} from 'plow-js';
 
+@connect($transform({
+    interfaceLanguage: $get('user.preferences.interfaceLanguage')
```

Comment:
Like in https://github.com/neos/neos-ui/pull/1535, please move this to the wrapping component (`neos-ui-editors`)